### PR TITLE
fd: add Chinese translation

### PR DESCRIPTION
**Pull request by Promix953**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/fd.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`fd "{{字符串|正则表达式}}"`
+
+- 查找以 `foo` 开头的文件：
+
+`fd "^foo"`
+
+- 查找具有特定扩展名的文件：
+
+`fd --extension txt`
```
Comment (einverne) [inline pages.zh/common/fd.md:17]:
```suggestion
`fd --extension {{txt}}`
```

File: pages.zh/common/fd.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+> 旨在比 `find` 更快且更易于使用。
+> 更多信息：<https://github.com/sharkdp/fd>.
+
+- 递归查找当前目录中匹配特定模式的文件：
+
+`fd "{{字符串|正则表达式}}"`
+
+- 查找以 `foo` 开头的文件：
+
+`fd "^foo"`
```
Comment (einverne) [inline pages.zh/common/fd.md:13]:
```suggestion
`fd "{{^foo}}"`
```

**Review by einverne — CHANGES_REQUESTED** [review]
Hi @Promix953  thanks for the new translation. I left a comment for you.

Can you help fix the original English version according to this [guideline](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).

**Comment by Promix953** [comment]
> Hi @Promix953 thanks for the new translation. I left a comment for you.嗨，感谢新的翻译。我为您留下了一条评论。
> 
> Can you help fix the original English version according to this [guideline](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).你能根据这个指南帮助修复原始英文版本吗？

#15038

**Review by einverne — APPROVED** [review]
LGTM.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**